db/migrate/20170329230309_create_requested_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRequestedTopics < ActiveRecord::Migration[5.0]
+  def change
+    create_table :requested_topics do |t|
```

Comment:
I like TopicRequest. Now that I think about it having a past tense is kind of implying too much about the class and doesn't give us flexibility to do other things with it in the future.